src/runtime_src/core/edge/ps_kernels/xrt/instance_query/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (C) 2022 Advanced Micro Devices, Inc. All rights reserved.
+#
+set(PS_KERNEL_INSTALL_DIR "${XRT_INSTALL_LIB_DIR}/ps_kernels_lib")
+
+add_library(instance_query OBJECT
+  instance_query.cpp
+  )
+
+set_target_properties(instance_query PROPERTIES 
+  VERSION ${XRT_VERSION_STRING}
+  SOVERSION ${XRT_SOVERSION}
+  )
+
+install (TARGETS instance_query 
+  EXPORT xrt-targets
+  LIBRARY DESTINATION ${PS_KERNEL_INSTALL_DIR}
+)
```

Comment:
Are you releasing an object file?   I am missing some context here.

Reply:
The purpose of this cmake is to create an object that can be inserted into an xclbin. This enables us to run the generated object as a ps kernel on a versal platform. Once the built-in PS kernel support is added I have no doubt this will be revisted. @rozumx Does this sound about right?

Reply:
Ah, got it.  This seems okay.
But I am not sure you should associate the target with an export as these object files are not really for external use via find_package?  I would try remove `EXPORT xrt-targets`.